src/screens/SendTokens.js

```diff
@@ -34,20 +38,52 @@ class SendTokens extends React.Component {
     // Holds the children components references
     this.references = [React.createRef()];
 
+    // Data pending to be sent while waiting ledger signatures
+    this.pendingData = null;
+
     /**
      * errorMessage {string} Message to be shown in case of error in form
      * loading {boolean} If should show spinner while waiting for server response
      * pin {string} PIN that user writes in the modal
      * txTokens {Array} Array of tokens configs already added by the user (start with only hathor)
+     * ledgerStep {number} When sending tx with ledger we have a step that needs user physical input, then we move to next step
      */
     this.state = {
       errorMessage: '',
       loading: false,
       pin: '',
-      txTokens: [hathorLib.constants.HATHOR_TOKEN_CONFIG]
+      txTokens: [hathorLib.constants.HATHOR_TOKEN_CONFIG],
+      ledgerStep: 0,
     };
   }
 
+  componentDidMount() {
+    if (IPC_RENDERER) {
+      IPC_RENDERER.on("ledger:txSent", (event, arg) => {
+        if (arg.success) {
+          this.getSignatures();
+        } else {
+          this.handleSendError(new LedgerError(arg.error.message));
+        } 
+      });
+
+      IPC_RENDERER.on("ledger:signatures", (event, arg) => {
+        if (arg.success) {
```

Comment:
Maybe we should refactor and move this block to a method.

Reply:
Done